### PR TITLE
testing/consolekit2: make consolekit2 replace consolekit if installed

### PR DESCRIPTION
**Pull request by PureTryOut**

### CODE REVIEW

**Comment by ncopa** [comment]
you need bump `pkgrel` if you modify the provides or other dependencies. The builders will not rebuild anything if unless `pkgver` or `pkgrel` changes.

**Comment by PureTryOut** [comment]
Good call, rookie mistake. Fixed and squashed.

File: main/consolekit/APKBUILD [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=consolekit
 pkgver=0.4.6
-pkgrel=2
+pkgrel=3
 pkgdesc="Framework for defining and tracking users, login sessions, and seats"
 url="http://www.freedesktop.org/wiki/Software/ConsoleKit"
 arch="all"
+provides="consolekit"
```
Comment (kaniini) [inline main/consolekit/APKBUILD]: consolekit itself doesn't need to provide consolekit, it's implied.

Comment (PureTryOut) [inline main/consolekit/APKBUILD]: Removed it

**Pull request closed without merging**